lib/Service/OpenAiAPIService.php

```diff
@@ -202,6 +359,9 @@ public function transcribe(?string $userId, string $audioFileContent, bool $tran
 	 * @throws \OCP\DB\Exception
 	 */
 	public function createImage(?string $userId, string $prompt, int $n = 1, string $size = Application::DEFAULT_IMAGE_SIZE): array {
+		if($this->isQuotaExceeded($userId, Application::QUOTA_TYPE_IMAGE)){
+			return ['error' => $this->l10n->t('Image generation quota exceeded')];
```

Comment:
Aaaah

Reply:
Fixed in: https://github.com/nextcloud/integration_openai/commit/6697be3b38618938a3fe9e5ab264d1eb7aeec0fe